gulpfile.js

```diff
@@ -53,7 +53,8 @@ var coreFiles = coreElems.map(function(n) { return path.src.core+n+".coffee"; }
 
 var extendElems = [
   "Easing", "GridCascade", "ParticleEmitter", "ParticleField", "QuadTree",
-  "SamplePoints", "StripeBound", "UI", "Noise", "Delaunay", "Shaping"
+  "SamplePoints", "StripeBound", "UI", "Noise", "Delaunay", "Shaping",
+  "ForceDirectedGraph", "Vertex", "Edge"
```

Comment:
Not sure if we should add `Vertex` and `Edge` as separate classes right now, unless they are useful enough on their own.